rbd-target-api.py

```diff
@@ -229,7 +229,7 @@ def get_targets():
     List targets defined in the configuration.
     **RESTRICTED**
     Examples:
-    curl --insecure --user admin:admin -X GET https://192.168.122.69:5000/api/targets
+    curl --insecure --user admin:admin -X GET http://192.168.122.69:5000/api/targets
```

Comment:
Nit: by changing "https" -> "http", that `--insecure` optional is no longer required (throughout) so perhaps just drop it.

Reply:
I am afraid it won't correct, the default it using the https:

`"api_secure": BoolSetting("api_secure", True),`
`http_mode = 'https' if settings.config.api_secure else 'http'`

Reply:
> I am afraid it is not correct, the default is using the https:
> 
> `"api_secure": BoolSetting("api_secure", True),`
> `http_mode = 'https' if settings.config.api_secure else 'http'`

Sorry, please ignore this. It works.

Reply:
Fixed it.